packaging/nmstate.spec

```diff
@@ -23,7 +23,7 @@ provider support on the southbound.
 
 %package -n python3-%{libname}
 Summary:        nmstate Python 3 API library
-Requires:       NetworkManager-libnm >= 1:1.22.10
+Requires:       NetworkManager-libnm >= 1:1.26.0
```

Comment:
Why is this needed? I am missing the reasoning in the commit message. Thanks!

Reply:
https://github.com/nmstate/nmstate/pull/1210#issuecomment-659952825

I don't think that's a good message to include in commit message

Reply:
Oh I didn't see that comment. Here is a suggestion:

```
NetworkManager 1.26 contains bug fixes needed by Nmstate.
```